src/Aspire.Dashboard/Otlp/Model/OtlpTrace.cs

```diff
@@ -96,6 +101,31 @@ static string BuildFullName(OtlpSpan existingSpan)
         }
     }
 
+    private static bool HasCircularReference(OtlpSpan span)
```

Comment:
Is this called every single time a span is sent to the OTEL endpoint?

Reply:
Yes